Examples/Io/Root/src/RootMaterialWriter.cpp

```diff
@@ -169,6 +170,15 @@ void ActsExamples::RootMaterialWriter::writeMaterial(
           rho->SetBinContent(b0 + 1, b1 + 1, mat.material().massDensity());
         }
       }
+    } else if (bins1 == 1 && bins0 == 1) {
+      // homogeneous surface
+      auto mat = sMaterial->materialSlab(Acts::Vector3{0, 0, 0});
+      t->SetBinContent(1, 1, mat.thickness());
+      x0->SetBinContent(1, 1, mat.material().X0());
+      l0->SetBinContent(1, 1, mat.material().L0());
+      A->SetBinContent(1, 1, mat.material().Ar());
+      Z->SetBinContent(1, 1, mat.material().Z());
+      rho->SetBinContent(1, 1, mat.material().massDensity());
```

Comment:
just for me to understand: what is that special treatment for?

Reply:
We are writing also the HomogeneousMaterial to a 1-1 binned root structure, in order to have identical trees.